packages/yourloops/test/integration/beforeEach.js

```diff
@@ -13,3 +13,5 @@ i18nInit(options).then(() => {
 }).catch((reason) => {
   console.error(reason)
 })
+
+jest.mock('@auth0/auth0-react')
```

Comment:
Since it's open to changes in the future (because for now mockAuth0Hook is mocking a HCP) I think each test should define if the user is a mocked HCP or a mocked User. I would keep it in the test for that.